attributes/default.rb

```diff
@@ -32,7 +32,7 @@
 
 default['consul']['service']['config_dir'] = join_path config_prefix_path, 'conf.d'
 
-default['consul']['version'] = '0.7.2'
+default['consul']['version'] = '0.7.3'
```

Comment:
Well, pushing out a change - was half way through.  Update should be here shortly on this.